docs/cloudshell-tutorial.md

```diff
@@ -0,0 +1,113 @@
+# Online Boutique
+
+##
+
+This tutorial shows you how to deploy the cloud-native microservices demo application **[Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo)** to a Kubernetes cluster.
+
+You'll be able to run Online Boutique on:
+
+- a free [minikube](https://minikube.sigs.k8s.io/docs/) cluster, which comes built in to the Cloud Shell instance
+- a [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) cluster using a new or existing [Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)
+
+Let's get started!
+
+## Kubernetes cluster setup
+
+Set up a Kubernetes cluster using the instructions below for either **minikube** or **GKE**.
+
+---
+### Minikube instructions
+
+Minikube creates a local Kubernetes cluster on Cloud Shell.
+
+1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+
+2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+
+3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+
+Once minikube has finished starting, you're ready to move on to the next step.
```

Comment:
nit: do you want to mention that starting minikube may take a few minutes?

Reply:
Good idea - added.